### PR TITLE
Review tests badge reports

### LINKED ISSUE #334: Review the numbers we show in the test badge
The testing badge at the top of the Readme is being generated here:
https://github.com/dylibso/chicory/blob/f916584fb6e73ee6399e5b94ef724dd0222dc67c/.github/workflows/ci.yaml#L89

I think that the numbers are a bit off after we added the `aot` module, we should check and eventually fix any finding.

### PR DESCRIPTION
**Pull request by andreaTP**
Hopefully this should fix #334 

I couldn't get the summary to work with the "comparison with previous commit" and, separating the summaries was the most straightforward.
You can check the result here:
https://github.com/andreaTP/chicory/actions/runs/9663987713

and this is an example badge generated: https://github.com/andreaTP/chicory/actions/runs/9663987713/artifacts/1636290791

### CODE REVIEW

**Review by bhelx — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Chicory Runtime
 
-[![Test Results](https://gist.githubusercontent.com/andreaTP/69354d1cc6cf23e4c3c4a9a8daf7ea15/raw/badge.svg)](https://gist.githubusercontent.com/andreaTP/69354d1cc6cf23e4c3c4a9a8daf7ea15/raw/badge.svg)
-[![](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://chicory.zulipchat.com)
+[![Interpreter Test Results](https://gist.githubusercontent.com/andreaTP/69354d1cc6cf23e4c3c4a9a8daf7ea15/raw/badge-interpreter.svg)](https://gist.githubusercontent.com/andreaTP/69354d1cc6cf23e4c3c4a9a8daf7ea15/raw/badge-interpreter.svg)
```
Comment (bhelx) [inline README.md:3]: No rush, but these links seem to be 404ing at the moment

Comment (andreaTP) [reply README.md:3]: Uhm, this seems to work in the Readme.

https://gist.githubusercontent.com/andreaTP/69354d1cc6cf23e4c3c4a9a8daf7ea15/raw/badge-interpreter.svg

Comment (andreaTP) [reply README.md:3]: Maybe a little hiccup on the gist side?

Comment (bhelx) [reply README.md:3]: yeah seems good now :)

**Pull request merged**